Sources/SwiftInspectorVisitors/Tests/FunctionDeclarationVisitorSpec.swift

```diff
@@ -216,10 +222,14 @@ final class FunctionDeclarationVisitorSpec: QuickSpec {
         it("finds the correct number of parameters") {
           expect(self.sut.functionDeclarations.first?.arguments?.count) == 2
         }
-        it("finds the parameter labels") {
+        it("finds the argument labels") {
```

Comment:
Can we add a test somewhere for a case where there is no argument label and only a parameter name?

Reply:
I don't think that's possible, the argument label is always present IIUC?

> Each function parameter has both an argument label and a parameter name. The argument label is used when calling the function; each argument is written in the function call with its argument label before it. The parameter name is used in the implementation of the function. 

I think you're referring to having the argument label be the same as the parameter name, which I think we already have tests for?

Reply:
I just pushed a new commit that hopefully will help clarify a bit https://github.com/fdiaz/SwiftInspector/pull/136/commits/f4c285dd18f85cbd4fd352cb0026d2286ea6e77e